controller/konnect/handle_status_conditions.go

```diff
@@ -25,32 +25,40 @@ func handleAPIAuthStatusCondition[T interface {
 	ent T,
 	apiAuth konnectv1alpha1.KonnectAPIAuthConfiguration,
 	err error,
+	dependingConditions ...metav1.Condition,
```

Comment:
Can we add a godoc about this parameter so that it's clear what it's used for?

Reply:
Left a comment about it